cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1611,6 +1611,8 @@ if (testing)
                   -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}
                   -DCMAKE_CXX_FLAGS=${CMAKE_CXX_FLAGS}
                   -DCMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}
+                  -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY_DEBUG:PATH=""
+                  -DCMAKE_ARCHIVE_OUTPUT_DIRECTORY_RELEASE:PATH=""
```

Comment:
Won't this overwrite things in a multiconfiguration build?

Reply:
The goal is to have a single location for the libraries (i.e. preventing them to go in the 'Debug' or 'Release' sub-directories on Windows)

Reply:
Ok, thanks!